Cargo.toml

```
@@ -52,6 +52,7 @@ shuttle-actix-web = { version = "0.21.0", optional = true }
 shuttle-runtime = { version = "0.21.0", optional = true }
 shuttle-static-folder = { version = "0.21.0", optional = true }
 tokio = { version = "1.29.1", optional = true }
+uts2ts = "0.1.0"
```

```suggestion
uts2ts = "0.3.0"
```

I have changed the crate so that less memory is allocated for the struct. It's probably not a big win in regards to perf, but it doesn't hurt either.